Composer/packages/client/src/recoilModel/dispatchers/notification.ts

```diff
@@ -6,10 +6,10 @@ import { CallbackInterface, useRecoilCallback } from 'recoil';
 import { v4 as uuid } from 'uuid';
```

Comment:
I think currently a new notification is added to the end of the array, shouldn't it be added to the front of the array?

Reply:
@sangwoohaan